ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/influxdb.j2

```diff
@@ -0,0 +1,45 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - influxdb #}
+{% if influxdb is arista.avd.defined %}
```

Comment:
Sorry for missing this general issue. Please rename the top key (including the name of template and schema) to `monitor_telemetry_influx`. Thanks.

Reply:
Fixed.